tests/remote/test_fennec.py

```diff
@@ -17,7 +17,8 @@
     # -p android-api-9 --branch=mozilla-central
     {'args': {'application': 'fennec',
               'platform': 'android-api-9',
-              'branch': 'mozilla-central'}},
+              'branch': 'mozilla-central',
+              'date': '2016-01-01'}},
```

Comment:
I did not check for that. I only looked for a valid build... What is the significance of having the last build?